api/api.go

```diff
@@ -145,6 +155,16 @@ func (c *Config) ClientConfig(region, address string, tlsEnabled bool) *Config {
 		WaitTime:   c.WaitTime,
 		TLSConfig:  c.TLSConfig.Copy(),
 	}
+
```

Comment:
having a transport with timeout seems useful for external use of this client as well. would be nice to either use a default timeout here when timeout=0 or set it in the NewClient method below.

Reply:
Added a helper to the config that sets the timeout. This is being used by the helper to get the node client but it is as simple as:
```
d := DefaultConfig()
d.SetTimeout(t)
c := NewClient(d)
```

To get a timed out client now.